pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, bazel, cacert, enableNixHacks ? true }:
```

Comment:
how did you manage to eliminate the nix hacks?

Reply:
I didn't actually eliminate them, instead I force-enabled them by default for all Nix Bazel builds :D They are needed in the builder to ensure `NIX_*` environment variables always get passed.